inc/thorvg_capi.h

```diff
@@ -398,7 +398,7 @@ TVG_EXPORT Tvg_Result tvg_canvas_clear(Tvg_Canvas* canvas, bool free);
 * - TVG_RESULT_SUCCESS: if ok.
 * - TVG_RESULT_INVALID_PARAMETERS: if canvas is invalid]
 */
-TVG_EXPORT Tvg_Result tvg_canvas_update(Tvg_Canvas* canvas);
+TVG_EXPORT Tvg_Result tvg_canvas_update(Tvg_Canvas* canvas, bool force);
```

Comment:
User have no idea what happens update() internally, it brings user wonder why force is necessary.
I think thorvg can conditionally make it in smarter way.

ex)
1. CanvasImple could have a boolean such as forceUpdateAll;
2. when target is changed, it set forceUpdateAll = true
3. canvas update() is called, override flag with forceUpdateAll update paints 
4. reset forceUpdateAll in draw() after drawing paints.

Reply:
Please check #229